srcs/requirements/server/public/src/app.ts

```
@@ -0,0 +1,56 @@
+ */
+async function loadPage(page: string): Promise<void> {
+    const app = document.getElementById("app") as HTMLElement | null;
+
+    if (!app) {
+        console.log("Error: app not found");
+        return;
+    }
+
+    try {
+        // Fetch the page content
+        const response = await fetch("pages/" + page);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)